src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { PageLayout, Logo, ContractPanel } from "../components";
+import { Mesh } from "../contracts/Mesh";
+import { Mesh__factory } from "../contracts/factories";
+import { useWagmi } from "../hooks";
+
+const Dashboard: React.FC = () => {
+  const wagmi = useWagmi();
+  const [meshie, setMeshie] = useState<Mesh>();
+  const [address, setAddress] = useState<string>();
+
+  useEffect(() => {
+    if (!wagmi.signer || !wagmi.address) return;
+
+    if (!process.env.VITE_PROXY_ADDRESS)
+      throw new Error("Error: no proxy address in env!");
```

Comment:
maybe `cannot find environment variable VITE_PROXY_ADDRESS`, just for clarity

Reply:
👍